driver-core/src/main/java/com/datastax/driver/core/Metadata.java

```diff
@@ -404,7 +413,7 @@ static String handleId(String id) {
             return id.toLowerCase();
 
         // Check if it's enclosed in quotes. If it is, remove them and unescape internal double quotes
-        if (id.charAt(0) == '"' && id.charAt(id.length() - 1) == '"')
+        if (!id.isEmpty() && id.charAt(0) == '"' && id.charAt(id.length() - 1) == '"')
```

Comment:
Small enhancement backported from 3.0: this line would throw an OutOfBoundsException for the empty string.